test/fixtures/box/missingYMinMaxReverse.js

```diff
@@ -0,0 +1,45 @@
+module.exports = {
+  tolerance: 0.0055,
+  config: {
+    type: 'scatter',
+    options: {
+      scales: {
+        x: {
+          display: false,
+          labels: ['January', 'February', 'March', 'April', 'May', 'June', 'July'],
+        },
+        y: {
+          display: true,
+          reverse: true,
+          min: -10,
+          max: 50
+        }
+      },
+      plugins: {
+        annotation: {
+          annotations: {
+            first: {
+              type: 'box',
+              yMax: 1,
```

Comment:
Did you need to change the order of annotations for reversed scale? I would not expect that.

Reply:
No, I didn't. I had tested both sequence and I left the last one. I'm committing having the same config.

Reply:
it should be done